### ISSUE TITLE
Lots of GTK warnings kill my terminal

### ISSUE THREAD

**Issue by somospocos** [EliverLara/Nordic#53]
When I'm starting gvim from a terminal, I get a lot of GTK garbage warnings that effectively kill my terminal (I use terminator). Output example:

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5261:14: 'min-height' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5294:26: Missing name of pseudo-class

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5395:50: Missing name of pseudo-class

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5398:52: Missing name of pseudo-class

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5410:14: 'min-height' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5489:11: 'min-width' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5490:12: 'min-height' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5502:12: 'min-height' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5503:11: 'min-width' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5556:21: '-gtk-outline-radius' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5703:18: '-gtk-icon-shadow' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5714:18: '-gtk-icon-shadow' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5970:35: Missing name of pseudo-class

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5973:37: Missing name of pseudo-class

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:5976:37: Missing name of pseudo-class

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:6014:24: Missing name of pseudo-class

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:6036:18: '-gtk-icon-shadow' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:6044:20: '-gtk-icon-shadow' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:6052:20: '-gtk-icon-shadow' is not a valid property name

    (gvim:1353): Gtk-WARNING **: Theme parsing error: gtk.css:6054:26: Missing name of pseudo-cl

I am using the latest stable release Nordic-1.6.5

**Comment by EliverLara** [comment]
Hi, I haven't updated the releases on github, you can download the latest version on [gnome-look](https://www.pling.com/p/1267246/) or download the [master zip](https://github.com/EliverLara/Nordic/archive/master.zip) directly from github.

**Comment by somospocos** [comment]
Do you have any plans to make a new release on github? Because I have no idea how to get those four zip files containing four different variations of the theme from master.zip or the git repository.

**Comment by somospocos** [comment]
Ok, I got the files from pling, set the theme to **Nordic-blue-standard-buttons** and still getting a lot of warnings. This time I went deeper and I was able to find the root of the problem. It wasn't terminator, it was tmux running inside it. By default tmux keeps a history of 2000 lines, according to its manpage. Running gvim with that theme produced almost 2600 lines of warnings and killed tmux. I was able to mitigate the issue by increasing the **history-limit** value in **~/.tmux.conf**.
